### PR TITLE
refactored and enhanced by international measures

### PR DESCRIPTION
**Pull request by rsimai**

### CODE REVIEW

File: recipes/main/lasagne_bolognaise.adoc [hunk 1]
```diff
@@ -15,69 +15,80 @@ image::lasagne_bolognaise/IMG_0835.jpg[Lasagne Bolognaise, 640, 480]
...
+|     |           | Olive Oil | Olivenöl
+|=========================================================
+
+
+.Béchamel
+[width="100%",cols="3,3,10,10"]
+|=========================================================
+|800ml | 27 fl oz | Milk | Milch
+| 60g  | 2 oz | Butter | Butter
+| 3 heaped tbsp |  | Wheat Flour | Weizenmehl
```
Comment (dcermak) [inline recipes/main/lasagne_bolognaise.adoc]: 1 tbsp is fyi 15ml :wink:

Comment (rsimai) [reply recipes/main/lasagne_bolognaise.adoc]: I doubt that, when it comes to "ein gehäufter Esslöffel Mehl" but found another reference that says "15g" :-)

Comment (rsimai) [reply recipes/main/lasagne_bolognaise.adoc]: please check again

**Review by aosthof — APPROVED** [review]
LGTM. Great idea about using a template!

**Pull request merged**